apps/webcomponents/src/app/components/gn-results-list/gn-results-list.html

```diff
@@ -1,4 +1,5 @@
 <gn-ui-results-list-container
   [layout]="layout"
   (mdSelect)="onMdClick($event)"
+  [showMore]='fixed ? "none" : "auto"'
```

Comment:
we could change the api and let people choose the `showMore` option

Reply:
yep that would be nice! didn't want to change the API in this PR though

Reply:
Done in the latest commit!